Documentation/developer-guide.md

```diff
@@ -23,6 +23,28 @@ We use the following workflow for building and testing features and fixes.
 
 You first need to [Fork](https://github.com/dotnet/corefx/wiki/Checking-out-the-code-repository#fork-the-repository) and [Clone](https://github.com/dotnet/corefx/wiki/Checking-out-the-code-repository#clone-the-repository) this WPF repository. This is a one-time task.
 
+
+### Running DRTs locally ###
+In order to run the set of DRTs on your local machine, pass the `-test` parameter to the `build.cmd` script. At the end of the run, you should see something like this:
+
+```
+  A total of 1 test Infos were processed, with the following results.
+   Passed: 1
+   Failed (need to analyze): 0
+   Failed (with BugIDs): 0
+   Ignore: 0
+
+```
+If there were any failures, you can cd into $(RepoRoot)\artifacts\test\$(Configuration)\$(Platform)\Test and run the tests manually with the `/debugtests` flag. This will open a cmd window, where you can open the test executable in Visual Studio and debug it. For example, to debug DrtXaml.exe you would enter this into the command window:
```

Comment:
This doesn't really tell anyone how to run the tests.  You should talk about the cmd scripts and what they do (e.g. RunDrts, etc.)  For instance, it can be confusing to run `RunDrtsDebug` and not realize you are debugging QV and not the tests.